core/src/main/java/uk/ac/bbsrc/tgac/miso/core/data/SequencingParameters.java

```diff
@@ -143,4 +144,32 @@ public void setUpdatedBy(User updatedBy) {
     this.updatedBy = updatedBy;
   }
 
+  public static Comparator<SequencingParameters> nameComparator = (sp1, sp2) -> sp1.compareTo(sp2);
+
+  @Override
+  public int compareTo(SequencingParameters other) {
+    // v4 2x126, v4, Rapid Run, 10X, Custom, v3, other
+    if (!(other instanceof SequencingParameters)) return -1;
```

Comment:
Is this meant to be a null check? If non-null, `other` has to be an instance of `SequencingParameters`

Reply:
Right sorry, too much copypasta. I'll remove this line.